tools/packaging/rpm/Dockerfile.build

```diff
@@ -1,14 +1,19 @@
 FROM centos:7
 
 RUN yum upgrade -y && \
+    yum remove git && \
+    yum install -y https://centos7.iuscommunity.org/ius-release.rpm && \
     yum install -y epel-release centos-release-scl && \
-    yum install -y fedpkg golang sudo make which cmake3 \
+    yum install -y sudo git2u fedpkg make which cmake3 \
                    automake autoconf autogen libtool \
                    devtoolset-6-gcc devtoolset-6-gcc-c++ \
                    devtoolset-6-libatomic-devel ninja-build && \
+    rpm --import https://mirror.go-repo.io/centos/RPM-GPG-KEY-GO-REPO &&\
```

Comment:
I'm not liking go-repo.io as the cononical source of golang compilation in binaries Istio produces...

IMO Red Hat should be the conical source of this repo.  The usage of `COPR` would be fine to reduce the red tape..